nilearn/image/image.py

```diff
@@ -932,6 +932,59 @@ def math_img(formula, **imgs):
     return new_img_like(niimg, result, niimg.affine)
 
 
+def bin_img(img, threshold=0, mask_img=None):
+    """Binarize an image such that its values are either 0 or 1.
+
+    .. versionadded:: 0.8.1
+
+    Parameters
+    ----------
+    img : a 3D/4D Niimg-like object
+        Image which should be binarized.
+
+    threshold : :obj:`float` or :obj:`str`
+        If float, we threshold the image based on image intensities meaning
+        voxels which have intensities greater than this value will be kept.
+        The given value should be within the range of minimum and
+        maximum intensity of the input image.
+        If string, it should finish with percent sign e.g. "80%" and we
+        threshold based on the score obtained using this percentile on
+        the image data. The voxels which have intensities greater than
+        this score will be kept. The given string should be
+        within the range of "0%" to "100%".
+
+    mask_img : Niimg-like object, default None, optional
+        Mask image applied to mask the input data.
+        If None, no masking will be applied.
+
+    Returns
+    -------
+    :class:`~nibabel.nifti1.Nifti1Image`
+        Binarized version of the given input image.
```

Comment:
maybe indicate that the return type is int.

Reply:
fixed